### DISCUSSION TITLE
cli Interface

### DISCUSSION THREAD

**Discussion by druskus20** [athensresearch/athens#1052] [Feature Requests]
One of the features I miss the most is a way to interact with Athens from the command line. This could be extremely useful when it comes to scripting missing functionality, integration with other tools / editors or even compatibility. Besides, I myself would often preffer to use Athens in a cli environment.
A cli interface could range from just entry management, creating and deleting notes etc, to a rich tui tool such as the one that Joplin has. 
In addition, I also find interesting the idea of an emacs / vim plugging to write and manage notes.

**Comment by hoopahmadness** [comment]
I would also love to do this; I want to be able to write my own code that can automatically view/update notes inside of Athens but I can't make heads or tails of the format of the .transit file. A CLI tool with the ability to append or delete a note with a given title would be fantastic.
